pip/update.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+import os
+
+update_list = []
+p = os.popen('sudo -H pip freeze --local',"r")
```

Comment:
As a personal script, this is fine but you should probably use ```subprocess.Popen``` for production scripts.